.storybook/components/SliderStory.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Slider from '../../components/Slider';
+import TextInput from '../../components/TextInput';
+
+storiesOf('Slider', module)
+  .addWithInfo(
+    'default',
+    `
+      lorem
+    `,
+    () => (
+      <div style={{ marginTop: '2rem' }}>
+        <div className="bx--form-item">
+          <label htmlFor="slider" className="bx--label">Slider Label</label>
+          <Slider id="slider" value={50} min={0} max={100} step={1} onChange={(evt) => { console.log(evt)}}>
```

Comment:
I think it would be good to offer 2 variants of `Slider`: one with a label, and one without.
In my opinion, it's something we should do for all of our Form elements. You're already providing a variant without the label, so making a variant with a label can help to ensure that a user can give an `id` prop value once and it'll be used for `id` and `htmlFor`.